ibis/omniscidb/client.py

```diff
@@ -488,6 +488,62 @@ def alter(self, tbl_properties=None):
         """
         raise NotImplementedError('This method is not implemented yet!')
 
+    def insert(
+        self,
+        obj: Optional[Union[pd.DataFrame, ir.Expr]] = None,
+        dst_cols: Optional[list] = None,
+        values: Optional[list] = None,
+        validate: Optional[bool] = True,
+    ):
+        """
+        Insert data into OmniSciDB table.
+
+        Parameters
+        ----------
+        obj : pd.DataFrame or ibis Expr, optional
+            Set obj to pandas dataframe or ibis Expr. Default is None
+        dst_cols : list, optional
+            Set list of table's column(s) into which data will be coppied
+            when ibis Expr or values is passed. Default is None
+        values : list, optional
+            Set list of values. Default is None
```

Comment:
This description doesn't add any value. If you don't have anything to explain, please don't just write things that are obvious from the name of the variable and type. Please explain what it is expected in the descriptions of the parameters, so a user with no knowledge of this functions knows how this value will be used.